addon/utils/ce/editor/update.js

```diff
@@ -380,11 +382,23 @@ function updateInnerContent(domNodes, {remove, set}) {
       domNode.innerHTML = '';
     }
     if (operationWantsToSetInnerHTML(set)) {
-      domNode.innerHTML = set.innerHTML;
+      domNode.innerHTML = sanitizeHTML(set.innerHTML);
     }
   }
 }
 
+/**
+ * Sanitizes the html before inserting it
+ * @method sanitizeHTML
+ */
+function sanitizeHTML(html) {
+  const safeAttributes = ['colspan', 'rowspan', 'title', 'alt', 'cellspacing', 'axis', 'about', 'property', 'datatype', 'typeof', 'resource', 'rel', 'rev', 'content', 'vocab', 'prefix', 'href', 'src', 'style', 'class'];
```

Comment:
Yes, it does that, but I don't know how to create the API for the user, because if I allow only to add safeAttributes instead of defining the list the user won't be able to remove attributes from that list, I know is not the best user experience.
What do you suggest?